runners/apex/src/test/java/org/apache/beam/runners/apex/examples/StreamingWordCountTest.java

```diff
@@ -110,12 +113,26 @@ public void testWindowedWordCount() throws Exception {
           && FormatAsStringFn.RESULTS.containsKey("bar")) {
         break;
       }
-      Thread.sleep(1000);
+      result.waitUntilFinish(Duration.millis(1000));
     }
     result.cancel();
     Assert.assertTrue(
         FormatAsStringFn.RESULTS.containsKey("foo") && FormatAsStringFn.RESULTS.containsKey("bar"));
     FormatAsStringFn.RESULTS.clear();
 
   }
+
+  @Ignore
+  @Test
+  public void testWordCountExample() throws Exception {
```

Comment:
It would be a circular dependency. Examples depend on runner, runner depends on examples. To get there, the example pipelines should be separate from the integration test project.